cypress/integration/publishingFlows/postBrowserPersistence.spec.js

```diff
@@ -0,0 +1,285 @@
+describe('Post Editor', () => {
+  function getPostContent() {
+    return cy.get('@articleForm').findByLabelText(/^Post Content$/i);
+  }
+
+  describe('v1 Editor', () => {
+    beforeEach(() => {
+      cy.testSetup();
+      cy.fixture('users/articleEditorV1User.json').as('user');
+
+      cy.get('@user').then((user) => {
+        cy.loginUser(user).then(() => {
+          cy.visit('/new');
+        });
```

Comment:
```suggestion
        cy.loginAndVisit(user, '/new');
```
suggestion (not blocking) - could make use of one of the new cypress commands here 😄  It's not essential, but given we have two different users in this set of tests, I think it's worth being extra careful that network side effects are accounted for